client/src/pages/Login.js

```diff
@@ -0,0 +1,126 @@
+import React, { Component } from "react";
+import TextField from '@material-ui/core/TextField';
+import Grid from '@material-ui/core/Grid';
+import '../App.scss';
+import Button from '@material-ui/core/Button';
+
+
+const initalState = {
+  email: "",
+  emailError: "",
+  password: "",
+  passwordError: "",
+}
+
+class LoginPage extends Component {
+  state = initalState;
+
+
+  handleEmailChange = event => {
+    this.setState({ email: event.target.value });
+  };
+
+  handlePasswordChange = event => {
+    this.setState({ password: event.target.value });
+  };
+
+
+  validate = () => {
+    let emailError = "";
+    // let passwordError = ""; 
+
+    if (!this.state.email.includes('@')) {
+      emailError = 'Invalid email';
+    }
```

Comment:
Something like 'bla@bla' would pass. I would use something like a regex or a 'validator' package, it's very easy to use.

Reply:
Done